src/config.js

```diff
@@ -5,6 +5,7 @@ export const localStorageFlowchartLink = 'KedroViz-link-to-flowchart';
 export const localStorageShareableUrl = 'KedroViz-shareable-url';
 export const localStorageFeedbackSeen = 'KedroViz-feedback-seen';
 export const localStorageBannerStatus = 'KedroViz-banners';
+export const localStorageLastRunEndTime = 'KedroViz-last-run-end-time';
```

Comment:
can we name it so its more specific for workflow view? something like `KedroViz-workflow-last-run-end-time`